streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordCollectorImpl.java

```diff
@@ -212,6 +227,7 @@ public <K, V> void send(final String topic,
 
             if (exception == null) {
                 final TopicPartition tp = new TopicPartition(metadata.topic(), metadata.partition());
+                log.info("Produced key:{}, value:{} successfully to tp:{}", key, value, tp);
```

Comment:
This seems to produce a lot of large log messages. Could be related to a recent performance regression that we have seen. Was this left here on purpose @vamossagar12 ?

Reply:
that's a nice find. No it's not on purpose and is a miss.